### DISCUSSION TITLE
求问context length exceeded的问题

### DISCUSSION THREAD

**Discussion by piliwu** [MuiseDestiny/zotero-gpt#109] [Q&A]
看一篇中文文献，只有10页，但是点击AskPDF后显示context_length_exceeded
ivalid_request_error
This model's maximum context length is 4097 tokens. However, your messages resulted in 5616 tokens. Please reduce the length of the messages.
我又试了40多页的英文文献，就没有这个问题。请问这要怎么解决？谢谢

**Comment by MuiseDestiny** [comment]
等后续更新修复吧。因为部分论文解析出来的段落太长。需要进一步分割。这个后面会做
